client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/HotRodConstants.java

```diff
@@ -42,7 +42,8 @@
    static final byte ADD_CLIENT_LISTENER_REQUEST = 0x25;
    static final byte REMOVE_CLIENT_LISTENER_REQUEST = 0x27;
    static final byte SIZE_REQUEST = 0x29;
-
+   static final byte EXEC_REQUEST = 0x2B;
```

Comment:
Should this be SCRIPT_REQUEST ? I noticed this was named EXEC all over the place.